src/quoridor/storage/storage.go

```diff
@@ -0,0 +1,20 @@
+package storage
+
+import (
+	"strconv"
+
+	"github.com/patrickmn/go-cache"
+)
+
+var c *cache.Cache
+
+func new() {
```

Comment:
Please use an uppercase for first letter, it's a convention in Go.

You can keep this method for further changes (use postgre client ...) for the moment. But please rename it it "init" or another name.

Reply:
if the first letter is in uppercase means that it is accessible from outside of the file.